src/components/GlobalPreferences/Notifications/Subscriptions.js

```diff
@@ -0,0 +1,200 @@
+import React, { useState, useCallback, useEffect } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { AppType } from '../../../prop-types'
+import {
+  Box,
+  DropDown,
+  GU,
+  IdentityBadge,
+  Info,
+  LoadingRing,
+  breakpoint,
+  font,
+  theme,
+} from '@aragon/ui'
+import { getSubscriptions } from './notification-service-api'
+
+export default function Subscriptions({ apps, email, token }) {
+  const [apiError, setApiError] = useState(null)
+  const [isFetching, setIsFetching] = useState(true)
+  const [subscriptions, setSubscriptions] = useState([])
+
+  useEffect(() => {
+    getSubscriptions(token)
+      .then(subscriptions => {
+        setSubscriptions(subscriptions)
+        setIsFetching(false)
+        return subscriptions
+      })
+      .catch(error => {
+        setIsFetching(false)
+        setApiError(error.message)
+      })
+  }, [email, token])
+
+  return (
+    <Box heading="Email notifications">
+      {apiError && <Info mode="error">Error verifying: {apiError}</Info>}
+      {isFetching && <LoadingRing />}
+      <SubscriptionsForm apps={apps} />
+      {!isFetching && <SubscriptionsTable subscriptions={subscriptions} />}
+    </Box>
+  )
+}
+
+Subscriptions.propTypes = {
+  apps: PropTypes.arrayOf(AppType).isRequired,
+  email: PropTypes.string,
+  token: PropTypes.string,
+}
+
+function SubscriptionsForm({ apps }) {
+  console.log(apps)
+  const subscriptionApps = apps.filter(
+    ({ isAragonOsInternalApp }) => !isAragonOsInternalApp
+  )
+
+  const appNames = ['', ...subscriptionApps.map(app => app.appName)]
+
+  const [selectedApp, setSelectedApp] = useState(0)
+  const [selectedEvent, setSelectedEvent] = useState(0)
+
+  const handleAppChange = (index, items) => {
+    setSelectedApp(index)
+    // TODO: update the events
+  }
+
+  const handleEventChange = (index, items) => {
+    setSelectedEvent(index)
+    // TODO: update the events
+  }
+
+  return (
+    <React.Fragment>
+      <Label>App</Label>
+      <DropDown
```

Comment:
The new `DropDown`'s api has changed a bit: `active` -> `selected` and it has a new `placeholder` you can use if an item has not been selected